docs/8-veba-troubleshooting.md

```diff
@@ -0,0 +1,232 @@
+# vCenter Event Broker Appliance - Troubleshooting
+
+## Requirements
```

Comment:
Some of the troubleshooting commands described, e.g. logs, can be done remotely via `faas-cli logs`. Could this be an option/alternative to evaluate?

Reply:
Added an example